### PR TITLE
test: check if default bucket object quota is applied

### PR DESCRIPTION
**Pull request by alimaredia**
Resolves: rhbz#1391500

Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

**Comment by alimaredia** [comment]
@andrewschoen once this is merged I will cherry-pick this commit and the other rgw testing related one to stable-2.2

File: tests/functional/tests/rgw/test_rgw_tuning.py [hunk 1]
```diff
@@ -3,9 +3,14 @@
 
 class TestRGWs(object):
 
-    def test_rgw_bucket_default_quota_is_set(self, node, File):
+    def test_rgw_bucket_default_quota_is_set(self, node, File, Command):
+        assert node["vars"]["copy_admin_key"] is True
         assert File(node["conf_path"]).contains("rgw override bucket index max shards")
         assert File(node["conf_path"]).contains("rgw bucket default quota max objects")
+        cmd = "sudo radosgw-admin user create --uid=test --display-name Test"
```
Comment (andrewschoen) [inline tests/functional/tests/rgw/test_rgw_tuning.py]: Do you need ``--cluster={{cluster}}`` here?

Comment (alimaredia) [inline tests/functional/tests/rgw/test_rgw_tuning.py]: looks like it, I'll give that a try

File: tests/functional/tests/rgw/test_rgw_tuning.py [hunk 2]
```diff
@@ -3,9 +3,14 @@
 
 class TestRGWs(object):
 
-    def test_rgw_bucket_default_quota_is_set(self, node, File):
+    def test_rgw_bucket_default_quota_is_set(self, node, File, Command):
```
Comment (andrewschoen) [inline tests/functional/tests/rgw/test_rgw_tuning.py]: Seems like you need ``@pytest.mark.no_docker`` here.

**Comment by andrewschoen** [comment]
jenkins test jewel-ansible2.2-docker_cluster

**Comment by leseb** [comment]
@andrewschoen are we good to go with this one?

**Pull request merged**